DevelopGitHub/GitHubExtension/GitHubExtension.Statistics.WebApi/CommunicationModels/GraphModel.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections.Generic;
+
+namespace GitHubExtension.Statistics.WebApi.CommunicationModels
+{
+    public class GraphModel
+    {
+        public GraphModel()
```

Comment:
Do you really need model which holds just collections of objects? I suggest may be you can send multiple requests to server to get all parts of this model. May be you don't needed such monolitic model?